pkg/app/web/src/components/header/index.tsx

```diff
@@ -192,6 +192,9 @@ export const Header: FC = memo(function Header() {
         id="more-menu"
         anchorEl={moreAnchorEl}
         open={Boolean(moreAnchorEl)}
+        getContentAnchorEl={null}
```

Comment:
nits, do we need to pass `null` here? I mean if we just pass null value, then looks like it's necessary to mention it on the caller side.

Reply:
Firstly I do not pass `null`, but it does not works well. So I pass `null` ( I referred [this stackoverflow](https://stackoverflow.com/questions/48157863/how-to-make-a-dropdown-menu-open-below-the-appbar-using-material-ui) )

> it's necessary to mention it on the caller side.

What does this mean?

Reply:
sorry, it's typo. I missed `not` word 😓

Reply:
I see, thanks for the explanation. Looks like someone had the same idea to make that null value as a default value, not to set from the caller side, but somehow is not available on our current using version, maybe it's way outdated already 😂 
Let's keep it as is, thank you 👍 

ref: https://github.com/mui/material-ui/issues/7961#issuecomment-442016178